contrib/storage-kafka/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Please remove. Now we have common logging configuration for all in drill-common module.